### DISCUSSION TITLE
Any guide to use an SSD1351 128X128 of DfRobot.

### DISCUSSION THREAD

**Discussion by Gfgo** [olikraus/u8g2#2128] [Q&A]
Hi to every one, I resently discover this awsome library and i'm working with an oled screen from Dfrobot 

Working Voltage：3.3V，5V
262,144 Colors(max)
128×128 RGB pixel resolution
Interface：SPI or Gadgeteer
temperature：-30℃ ~ +70℃
OLED Size：26.855 × 26.864(mm)
Module Size：52.00 × 42.00(mm)
Weight：20 g
Size：1.5 inch
Driver IC：SSD1351

I use the ssd1351 control fom u8glib (actually works) but, don't know wich one use in u8g2 I have prove others form the u8g2 library but no one works.

The next constructors only show me the page color screen

U8G2_SH1107_128X128_1_4W_HW_SPI u8g2(U8G2_R0, /* cs=*/ 23, /* dc=*/ 22, /* reset=*/ 2);

U8G2_UC1617_JLX128128_1_4W_SW_SPI u8g2(U8G2_R0, /* clock=*/ 15, /* data=*/ 16, /* cs=*/ 23, /* dc=*/ 22, /* reset=*/ 2);

**Comment by olikraus** [comment]
U8glib had been designed as lib for monochrome displays. Over time I added RGB support to u8glib, but I realized that this is wrong. U8g2 is a way back to a plain monochrome lib. So... It doesn't support RGB Displays.

**Comment by Gfgo** [comment]
Anyway thank you for the help and for the libraries,I go back to Uclib, on the other hand is there a way to open discussions in Uclib?

**Reply by olikraus** [reply]
You could create an issue, but I stopped working on ucglib
